editor-components/Image_gallery/image_gallery_4/image-gallery4.tsx

```diff
@@ -439,131 +495,187 @@ class ImageGallery4 extends BaseImageGallery {
                   >
                     {navItems?.map((item: NavItem, index: number) => {
                       if (!this.castToString(item.title)) return null;
                       return (
-                        <li
+                        <div
                           key={index}
                           className={this.decorateCSS("list-item")}
                         >
-                          <button
-                            className={
-                              activeNav === index
-                                ? this.decorateCSS("active")
-                                : ""
-                            }
-                            onClick={() => {
-                              this.switchNav(index);
-                            }}
-                          >
-                            {item.title}
-                          </button>
-                        </li>
+                          {this.castToString(item.title) && (
+                            <button
+                              className={`${this.decorateCSS("button")} ${activeNav === index ? this.decorateCSS("active") : ""}`}
+                              onClick={() => {
+                                this.handleSectionClick(index);
+                              }}
+                            >
+                              {item.title}
+                            </button>
+                          )}
+                        </div>
                       );
                     })}
-                  </ul>
+                  </div>
                 )}
                 {showActiveNavSubnavs && subnavItems?.length > 0 && (
-                  <ul
+                  <div
                     className={`${this.decorateCSS("subnav-list")} ${this.decorateCSS("hide-scrollbar")}`}
                   >
                     {(subnavItems || []).map(
                       (item: SubnavItem, index: number) => {
                         if (!this.castToString(item.title)) return null;
                         return (
-                          <li
+                          <div
                             key={index}
                             className={this.decorateCSS("list-item")}
                           >
-                            <button
-                              className={
-                                activeSubnav === index
-                                  ? this.decorateCSS("active")
-                                  : ""
-                              }
-                              onClick={() => {
-                                this.switchSubnav(index);
-                              }}
-                            >
-                              {item.title}
-                            </button>
-                          </li>
+                            {this.castToString(item.title) && (
+                              <button
```

Comment:
use base button

Reply:
This button for section titles so i don't need base button